_data/hosting_providers.json

```diff
@@ -1,4 +1,14 @@
 [
+    {
+    "name": "Pantheon",
+    "link": "https://pantheon.io/",
+    "category": "full",
+    "tutorial": "https://pantheon.io/docs/guides/launch/",
```

Comment:
just to check -- usually, if users need to go through a tutorial to turn on https for their site, that's what we would call "partial" support. "full" means that if they've set up a site at all, it'll have https. could you help me understand which is the case here?